README.md

```diff
@@ -27,6 +27,18 @@ To enable the YAML support with [pyyaml](http://pyyaml.org/):
 pip install j2cli[yaml]
 ```
 
+To enable [ansible filters](https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#filters):
+
+```
+pip install j2cli[ansible]
+```
+
+Or both with
+
+```
+pip install j2cli[all]
```

Comment:
for all , can we just run `pip install j2cli` ?

Reply:
Nope, those are extra requires. If you want to have them, the `[]` are required. `all` is just a manual convenience alias including all other extra requires.

Reply:
Even if there is a way to make ut possibke, ansi le is quite heavy, it should be opt-int not opt-out.

Reply:
makes sense